src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,54 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, T2> {
```

Comment:
According to `Type Parameter Naming Conventions` https://docs.oracle.com/javase/tutorial/java/generics/types.html it will be better to name `Pair<T, V>`
T2 is not a good name for a parametrized type.